Algorithms/Sequences/TribonacciNumbersSequence.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.Numerics;
+
+namespace Algorithms.Sequences;
+
+/// <summary>
+///     <para>
+///         Tribonacci numbers: a(n) = a(n-1) + a(n-2) + a(n-3) with a(0)=a(1)=a(2)=1.
+///     </para>
+///     <para>
+///         OEIS: https://oeis.org/A000213.
+///     </para>
+/// </summary>
+public class TribonacciNumbersSequence : ISequence
+{
+    public IEnumerable<BigInteger> Sequence
+    {
+        get
+        {
+            yield return 1;
+            yield return 1;
+            yield return 1;
+            BigInteger beforePrevious = 1;
+            BigInteger previous = 1;
+            BigInteger current = 1;
+            while (true)
+            {
+                var next = beforePrevious + previous + current;
+                beforePrevious = previous;
+                previous = current;
+                current = next;
+                yield return next;
```

Comment:
Please do the same as I suggest above to avoid ugly `yield return 1`. Maybe try using a buffer here to see if it looks better than `beforePrevious`, `previous`, `current` triad.

Reply:
Fixed with the latest commit.